src/build_tools/hooks_hydra.py

```diff
@@ -368,3 +371,19 @@ def pre_module_hook(self, *args, **kwargs):  # pylint: disable=unused-argument
     ############################
 
     self.cfg.enable_templating = en_templ
+
+
+def end_hook():
+    """Hook to run shortly before EasyBuild completes"""
+
+    logger = fancylogger.getLogger()
+    fancylogger.logToScreen(True, stdout=True)
+    fancylogger.setLogLevelInfo()
+
+    # submit Lmod cache job
+    if os.getenv('BUILD_TOOLS_RUN_LMOD_CACHE'):
```

Comment:
where does this var come from?

What happens if you run this outside a job?

Reply:
it's set in the .bashrc of vsc10001.
i wanted to provide some flexibility to turn it off if needed

Reply:
Why not let it be set by the `submit build` script?